lib/mi_bridges/driver.rb

```diff
@@ -124,7 +127,8 @@ def setup
 
     def find_page_klass
       APPLY_FLOW.detect do |page|
-        page::TITLE == page_title
+        page_title == page::TITLE ||
+          page_title.match?(page::TITLE)
```

Comment:
nice! Was wondering when we were going to have to deal with page names that include peoples' FIRST NAME